user_comments/contrib/is_core/cores.py

```diff
@@ -40,9 +41,20 @@ def get_objects(self):
         )
 
 
+class DetailCommentModelFormView(DetailModelFormView):
+
+    def get_readonly_fields(self):
+        return (
+            tuple(field for field in super().generate_fields() if field != 'comment')
+            if self.core.get_update_only_comments(self.request, obj=self.get_obj())
+            else super().get_readonly_fields()
```

Comment:
I also had to change this - the previous version with `generate_fields` made all the fields read-only. It did not take into account standard core/view/form with general update allowed.